src/fluent_linter/linter.py

```diff
@@ -406,6 +405,25 @@ def visit_ResourceComment(self, node):
             )
             return
 
+    def visit_Placeable(self, node):
+        if "PS01" in self.config and not self.config["PS01"]["disabled"]:
+            if (node.span.start + 2) != node.expression.span.start:
+                self.add_error(
+                    node,
+                    self.last_message_id,
+                    "PS01",
+                    f"Placeables should be preceded by exactly one space (`{{ {node.expression.id.name} }}`).",
+                )
+            if (node.span.end - 2) != node.expression.span.end:
+                self.add_error(
+                    node,
+                    self.last_message_id,
+                    "PS01",
+                    f"Placeables should be preceded by exactly one space (`{{ {node.expression.id.name} }}`).",
```

Comment:
Heh, this is my mistake, but...

```suggestion
f"Placeables should be followed by exactly one space (`{{ {node.expression.id.name} }}`).",
```